src/SecurityDetails/Converters/AssetAllocationsConverter.ts

```diff
@@ -96,48 +100,56 @@ export class AssetAllocationsConverter extends SecurityDetailsConverter {
         // Prepare table
 
         table.deleteColumns();
-        table.setColumn('AssetAllocations_Type');
-
-        // Add asset allocations to table
-
-        if (json.length) {
 
-            // Update table
+        if (!json.length) {
+            return;
+        }
+        if (json.length > 1){
+            isCompare = true;
+        }
+        // Create tables
+        for (const security of json) {
+            const id = security.Id,
+                isin = security.Isin,
+                assetAllocations = security.Portfolios[0].AssetAllocations,
+                assetAllocationsTypeStr =
+                'AssetAllocations_Type' + (isCompare? `_${id}` : '');
```

Comment:
Really nitpicking here, but I believe that there should be a space after variable when using ternary operator.
```suggestion
                'AssetAllocations_Type' + (isCompare ? `_${id}` : '');
```